task.sql

```diff
@@ -1,2 +1,6 @@
 # Write your SQL code for the database creation here. Good luck! 
-USE ShopDB; 
+USE ShopDB;
+
+CREATE INDEX ON Customers (Email);
```

Comment:
```suggestion
CREATE INDEX Email ON Customers (Email);
```
Need to add name of index

Reply:
Added